Makefile

```diff
@@ -437,3 +437,7 @@ check_diff:
 
 check_shebang:
 	./tools/check-shebang.sh
+
+act:
```

Comment:
We have "install_prerequisites" above.
Please move it there w/o rm -rf. We don't know where it will be called and what may be deleted.